examples/aks/aks_cluster_gitops/README.md

```diff
@@ -104,4 +116,10 @@ helm upgrade -i castai-kvisor castai-helm/castai-kvisor -n castai-agent \
 
 This example can also be used to import AKS cluster to Terraform which is already onboarded to CAST AI console through [script](https://docs.cast.ai/docs/cluster-onboarding#how-it-works).   
 For importing existing cluster follow steps 1-3 above and change `castai_node_configuration.default` Node Configuration name.
-This would allow to manage already onboarded clusters' CAST AI Node Configurations and Node Templates through IaC.
\ No newline at end of file
+This would allow to manage already onboarded clusters' CAST AI Node Configurations and Node Templates through IaC.
+
+## Note: Difference between CAST API key & CAST Cluster API Token
```

Comment:
I am not sure this is the case. Could you elaborate?

Reply:
TF flow you should use user API access key only to run Terraform, but not pass it to Castware helm charts. 
It means that using your user API access key Terraform makes calls to our API to provision our platform resources such as: onboarding cluster, creating/updating node configurations/templates, etc.

To make cluster specific operations using castware API Token should be used.

While it is possible to use the same API access key when configuring Castware components through TF it is an antipattern. And it may cause friction and downtime for customers when deleting such API keys.